### ISSUE TITLE
3 PIPE constants missing for win32pipe.CreateNamedPipe()

### ISSUE THREAD

**Issue by dreamflow** [mhammond/pywin32#1942]
this is a feature request for 3 PIPE constants , that are missing
for the for the paramters "openMode" and "pipeMode" of
[win32pipe.CreateNamedPipe()](https://mhammond.github.io/pywin32/win32pipe__CreateNamedPipe_meth.html)

* Expected behavior :
that these 3 PIPE constants , as defined for the WinAPI
function [CreateNamedPipeW()](https://docs.microsoft.com/en-us/windows/win32/api/namedpipeapi/nf-namedpipeapi-createnamedpipew) , are available via
```
win32pipe.PIPE_ACCEPT_REMOTE_CLIENTS
win32pipe.PIPE_REJECT_REMOTE_CLIENTS
win32pipe.FILE_FLAG_FIRST_PIPE_INSTANCE
```

* Steps to reproduce the problem.
```
import win32pipe
print( win32pipe.PIPE_ACCEPT_REMOTE_CLIENTS )
print( win32pipe.PIPE_REJECT_REMOTE_CLIENTS )
print( win32pipe.FILE_FLAG_FIRST_PIPE_INSTANCE )
```

* Version of Python and pywin32 :
pywin32 : 304
python 64bit : 3.10.5 , 3.9.10
os : Win10 64bit enterprize 21H2

**Comment by dreamflow** [comment]
thank you very much @kxrob and @mhammond
for this ultra fast pull request and merge .

**Comment by mhammond** [comment]
Fixed by #1946
